code/modules/mob_spawn/ghost_roles/space_roles.dm

```diff
@@ -87,19 +87,50 @@
 	shoes = /obj/item/clothing/shoes/laceup
 	l_pocket = /obj/item/stack/medical/bruise_pack
 
+//monkestation edits begin
 ///asteroid comms agent
 
 /obj/effect/mob_spawn/ghost_role/human/lavaland_syndicate/comms/space
 	you_are_text = "You are a syndicate agent, assigned to a small listening post station situated near your hated enemy's top secret research facility: Space Station 13."
 	flavour_text = "Monitor enemy activity as best you can, and try to keep a low profile. Monitor enemy activity as best you can, and try to keep a low profile. Use the communication equipment to provide support to any field agents, and sow disinformation to throw Nanotrasen off your trail. Do not let the base fall into enemy hands!"
-	important_text = "DO NOT abandon the base."
+	important_text = "DO NOT leave the base. You are not a field agent, and are not permitted to do their job for them."
+	spawner_job_path = /datum/job/lavaland_syndicate/space
 
-/obj/effect/mob_spawn/ghost_role/human/lavaland_syndicate/comms/space/Initialize(mapload)
+//Admin PM from-Kondafairydragon: not that... but HOW THE FUCK DID A LAVALAND SYNDICATE END UP IN SPACE. rather
+//overrides job title so that doesnt happen again bwoinks are scary
+
+/obj/effect/mob_spawn/ghost_role/human/lavaland_syndicate/comms/space/anderson //the last agent according to lore, spawns rarely, has some flavour text, should start blinded
+	name = "sleeper"
+	desc = "A standard medicinal sleeper used to treat small and major injuries alike. It appears to be locked up, and you can see a gas mask through the fogged-up windows..."
+	icon_state = "sleeper"
+	you_are_text = "You are a Syndicate reconnaisance agent who went blind in an accident..."
+	flavour_text = "Your painkillers have ran out, your memories are getting foggy, and all you can remember is the codename 'Anderson'. You were about to hurl yourself out into space, but you heard the intercom announce that the backup was getting unfrozen... maybe you'll see again?"
+	important_text = "DO NOT leave the base, you're inevitably doomed without it!"
+	outfit = /datum/outfit/lavaland_syndicate/comms/anderson
+
+/obj/effect/mob_spawn/ghost_role/human/lavaland_syndicate/comms/space/anderson/Initialize(mapload)
+	. = ..()
+	if(prob(85)) //only has a 15% chance of existing, otherwise it'll just be a regular (self-control) sleeper
```

Comment:
instead of doing this just make a spawner effect, all you need to do is replace this with an if-else on the effects Initialize()

Reply:
this was jury rigged from the original comms agent spawner code (which has that funky shit effect where it only spawns like 10% of the time)